src/WinRT.Runtime/ObjectReference.cs

```diff
@@ -26,6 +26,7 @@ abstract class IObjectReference : IDisposable
         private readonly IntPtr _thisPtr;
         private object _disposedLock = new object();
         private IntPtr _referenceTrackerPtr;
+        protected IntPtr _contextToken;
```

Comment:
How come we've moving this field here rather than leaving it just in `ObjectReferenceWithContext<T>`? This will increase the size of every single object wrapper by `sizeof(nint)` even though they have no context to carry around. Can we not keep the context just in the derived type

Reply:
That would require to make the new properties virtual. I suppose we can do that.

Reply:
Just to recap: didn't we say we should make this internal?